WeakAuras/GenericTrigger.lua

```diff
@@ -3520,6 +3520,13 @@ function GenericTrigger.SetToolTip(trigger, state)
     elseif (state.unit and state.unitDebuffIndex) then
       GameTooltip:SetUnitDebuff(state.unit, state.unitDebuffIndex);
       return true
+    elseif (state.tooltip) then
```

Comment:
Make state.tooltip the first case. If e.g. both unit and tooltip are there, tooltip should be preferred.